lib/nfc/nfc_device.c

```diff
@@ -650,6 +650,35 @@ bool nfc_device_load_mifare_df_data(FlipperFormat* file, NfcDevice* dev) {
     return parsed;
 }
 
+static bool nfc_device_save_bank_card_data(FlipperFormat* file, NfcDevice* dev) {
+    bool saved = false;
```

Comment:
How this is related to badusb changes?
Same for desktop files, why emv saving was added here?

Reply:
This is not supposed to be part of the pull request, I used those files to practice with the code base originally. I will remove this right now sorry about that. I don't know why github is taking in the latest commit I genuinely thought it would stick with the commit of when I opened the issue. I'm also having issues with assigning a pull request to a related issue (#1989) should I rename the PR with the issue ID in the title?

Reply:
> This is not supposed to be part of the pull request, I used those files to practice with the code base originally. I will remove this right now sorry about that. I don't know why github is taking in the latest commit I genuinely thought it would stick with the commit of when I opened the issue. I'm also having issues with assigning a pull request to a related issue (#1989) should I rename the PR with the issue ID in the title?

You can add issue id with # to PR description, just put it in What's new section
So it will be related to this PR and automatically will be closed when PR is merged